src/pages/about.astro

```diff
@@ -103,12 +103,41 @@ const members = filterInactiveMembers(await getMembers());
       <section>
         <h2>What payments are eligible?</h2>
 
-        <p
-          >Companies directly pay maintainers of Open Source projects they depend on, with no strings attached. Paying
-          developers to do specific work does not count towards the Pledge. Feel free to participate in <a
-          href="https://github.com/opensourcepledge/osspledge.com/issues/112">our ongoing discussion</a> on what counts
-          and what doesn't.</p
-        >
+        <p>The primary intent with the Pledge is <b>no-strings-attached payments
+        to community projects</b>. Start there.</p>
+
+        <ol>
+
+          <li>Open Source projects that your company controls are out of scope.
+            If you own the trademark, it doesn't count.</li>
+
+          <li>Open Source projects that exclusively benefit your company or
+            your company's own ecosystem are out of scope.</li>
+
+          <li>It's okay if you get non-exclusive logo placement or other
+            acknowledgement for your payments, or an hour or two of consulting
+            time with the project team, but anything beyond such minimal
+            benefits puts the payment out of scope.</li>
+
+          <li>Payments to foundations are generally in scope, though most
+            foundations today are not set up to pay maintainers (they focus
+            more on conferences, trademarks, etc.). We hope that the Pledge
+            will enable foundations to shift towards paying maintainers in the
+            future.</li>
+
+          <li>It's great if your company also actively participates in the
+            projects you fund, via code contributions, membership on an
+            advisory council or board, and the like.</li>
```

Comment:
I'm missing a "but the time spent on that does not count towards the pledge for now"

Reply:
Addressed in a separate point in d2c358e39fd198349031bf0405eeb8f07796805e.